app/src/models/status.ts

```diff
@@ -31,6 +32,7 @@ export enum AppFileStatusKind {
 export type PlainFileStatus = {
   kind:
     | AppFileStatusKind.New
+    | AppFileStatusKind.Untracked
```

Comment:
:fire: this line shouldn't be needed if we're also defining a `UntrackedFileStatus` enum. Here's the changes that I had to make to be able to drop this line:

```diff
diff --git a/app/src/lib/git/log.ts b/app/src/lib/git/log.ts
index cd52e22d1..ea7965959 100644
--- a/app/src/lib/git/log.ts
+++ b/app/src/lib/git/log.ts
@@ -4,6 +4,7 @@ import {
   AppFileStatusKind,
   PlainFileStatus,
   CopiedOrRenamedFileStatus,
+  UntrackedFileStatus,
 } from '../../models/status'
 import { Repository } from '../../models/repository'
 import { Commit } from '../../models/commit'
@@ -20,7 +21,7 @@ import {
 function mapStatus(
   rawStatus: string,
   oldPath?: string
-): PlainFileStatus | CopiedOrRenamedFileStatus {
+): PlainFileStatus | CopiedOrRenamedFileStatus | UntrackedFileStatus {
   const status = rawStatus.trim()
 
   if (status === 'M') {
diff --git a/app/src/models/status.ts b/app/src/models/status.ts
index 79583020a..da29e39be 100644
--- a/app/src/models/status.ts
+++ b/app/src/models/status.ts
@@ -32,7 +32,6 @@ export enum AppFileStatusKind {
 export type PlainFileStatus = {
   kind:
     | AppFileStatusKind.New
-    | AppFileStatusKind.Untracked
     | AppFileStatusKind.Modified
     | AppFileStatusKind.Deleted
 }
```